packages/charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.styles.ts

```diff
@@ -0,0 +1,114 @@
+import { IVerticalStackedBarChartStyleProps, IVerticalStackedBarChartStyles } from './VerticalStackedBarChart.types';
+
+export const getStyles = (props: IVerticalStackedBarChartStyleProps): IVerticalStackedBarChartStyles => {
+  const { className, theme, width, height, legendColor, shouldHighlight, href } = props;
+
+  const chartWidth = width! + 50;
+  const chartHeight = height! + 50;
+  const chartMargin = { left: 35, right: 0, top: 35, bottom: 0 };
+
+  return {
+    root: [
+      theme.fonts.medium,
+      className,
+      {
+        width: chartWidth
+      }
+    ],
+
+    chartLabel: [
+      {
+        textAlign: 'center',
+        ...theme.fonts.mediumPlus
+      }
+    ],
+
+    chart: [
+      {
+        width: chartWidth,
+        height: chartHeight
+      }
+    ],
+
+    xAxis: [
+      {
+        transform: `translate(${chartMargin.left}px, ${height - chartMargin.bottom}px)`
+      },
+      {
+        selectors: {
+          text: {
+            ...theme.fonts.tiny
+            // opacity: 1
```

Comment:
>// opacity: 1 [](start = 12, length = 13)

why commented, remove if it is unnecessary

Reply:
done